src/entity.hpp

```diff
@@ -199,6 +199,18 @@ class Entity
 	//--PUBLIC AMBIENT PARTICLE EFFECT SKILLS--
 	Sint32& particleDuration;
 
+	//--PUBLIC DOOR SKILLS--
+	Sint32& doorDir;
```

Comment:
We might start adding comments here as to what reference correlates to which skill. E.g. `Sint32& doorDir; //skill[0]`